Too few arguments to otr_on_message_send()

```
src/event/client_events.c:160:24: error: too few arguments to function ‘otr_on_message_send’
     gboolean handled = otr_on_message_send(chatwin, plugin_msg);
```
This appears with OTR, but without GPG support:
```
// OTR supported, PGP unsupported
#ifdef HAVE_LIBOTR
#ifndef HAVE_LIBGPGME
    gboolean handled = otr_on_message_send(chatwin, plugin_msg);
```

Actually this configuration (+otr -gpg) has more compilation errors:
```
src/event/client_events.c: In function ‘cl_ev_send_msg’:
src/event/client_events.c:160:24: error: too few arguments to function ‘otr_on_message_send’
     gboolean handled = otr_on_message_send(chatwin, plugin_msg);
                        ^
In file included from ./src/ui/ui.h:47:0,
                 from ./src/command/cmd_defs.h:40,
                 from ./src/plugins/plugins.h:38,
                 from src/event/client_events.c:42:
./src/otr/otr.h:75:10: note: declared here
 gboolean otr_on_message_send(ProfChatWin *chatwin, const char *const message, gboolean request_receipt);
          ^
src/event/client_events.c:162:20: error: too few arguments to function ‘message_send_chat’
         char *id = message_send_chat(chatwin->barejid, plugin_msg, oob_url);
                    ^
In file included from src/event/client_events.c:45:0:
./src/xmpp/xmpp.h:133:7: note: declared here
 char* message_send_chat(const char *const barejid, const char *const msg, const char *const oob_url,
       ^
src/event/client_events.c:164:9: error: too few arguments to function ‘chatwin_outgoing_msg’
         chatwin_outgoing_msg(chatwin, plugin_msg, id, PROF_MSG_PLAIN);
         ^
In file included from ./src/command/cmd_defs.h:40:0,
                 from ./src/plugins/plugins.h:38,
                 from src/event/client_events.c:42:
./src/ui/ui.h:136:6: note: declared here
 void chatwin_outgoing_msg(ProfChatWin *chatwin, const char *const message, char *id, prof_enc_t enc_mode,
      ^
src/event/client_events.c:116:14: error: variable ‘request_receipt’ set but not used [-Werror=unused-but-set-variable]
     gboolean request_receipt = FALSE;
              ^
```

--- a/src/event/client_events.c
+++ b/src/event/client_events.c
@@ -159,9 +159,9 @@ cl_ev_send_msg(ProfChatWin *chatwin, const char *const msg, const char *const oo
 #ifndef HAVE_LIBGPGME
     gboolean handled = otr_on_message_send(chatwin, plugin_msg);
     if (!handled) {
-        char *id = message_send_chat(chatwin->barejid, plugin_msg, oob_url);
+        char *id = message_send_chat(chatwin->barejid, plugin_msg, oob_url, request_receipt);
         chat_log_msg_out(chatwin->barejid, plugin_msg);
-        chatwin_outgoing_msg(chatwin, plugin_msg, id, PROF_MSG_PLAIN);
+        chatwin_outgoing_msg(chatwin, plugin_msg, id, PROF_MSG_PLAIN, request_receipt);
         free(id);
     }
 
@@ -177,12 +177,12 @@ cl_ev_send_msg(ProfChatWin *chatwin, const char *const msg, const char *const oo
     if (chatwin->pgp_send) {
         char *id = message_send_chat_pgp(chatwin->barejid, plugin_msg);
         chat_log_pgp_msg_out(chatwin->barejid, plugin_msg);
-        chatwin_outgoing_msg(chatwin, plugin_msg, id, PROF_MSG_PGP);
+        chatwin_outgoing_msg(chatwin, plugin_msg, id, PROF_MSG_PGP, request_receipt);
         free(id);
     } else {
-        char *id = message_send_chat(chatwin->barejid, plugin_msg, oob_url);
+        char *id = message_send_chat(chatwin->barejid, plugin_msg, oob_url, request_receipt);
         chat_log_msg_out(chatwin->barejid, plugin_msg);
-        chatwin_outgoing_msg(chatwin, plugin_msg, id, PROF_MSG_PLAIN);
+        chatwin_outgoing_msg(chatwin, plugin_msg, id, PROF_MSG_PLAIN, request_receipt);
         free(id);
     }
 
@@ -195,9 +195,9 @@ cl_ev_send_msg(ProfChatWin *chatwin, const char *const msg, const char *const oo
 // OTR unsupported, PGP unsupported
 #ifndef HAVE_LIBOTR
 #ifndef HAVE_LIBGPGME
-    char *id = message_send_chat(chatwin->barejid, plugin_msg, oob_url);
+    char *id = message_send_chat(chatwin->barejid, plugin_msg, oob_url, request_receipt);
     chat_log_msg_out(chatwin->barejid, plugin_msg);
-    chatwin_outgoing_msg(chatwin, plugin_msg, id, PROF_MSG_PLAIN);
+    chatwin_outgoing_msg(chatwin, plugin_msg, id, PROF_MSG_PLAIN, request_receipt);
     free(id);
 
     plugins_post_chat_message_send(chatwin->barejid, plugin_msg);

